src/main/java/calculator/calculate/CalculatorService.java

```diff
@@ -0,0 +1,21 @@
+package calculator.calculate;
+
+import calculator.input.Numbers;
+import calculator.input.Symbols;
+public class CalculatorService {
+
+    public CalculatorService() {
+    }
+
+    public int calculate(Numbers numbers, Symbols symbols) {
+        int value = numbers.getNumbers().get(0).getNumber();
+
+        for (int i = 0; i < symbols.getSymbols().size(); i++) {
+            Arithmetic arithmetic = Arithmetic.fromSymbol(symbols.getSymbols().get(i).getSymbol());
+            value = arithmetic.apply(value, numbers.getNumbers().get(i + 1).getNumber());
+        }
+
```

Comment:
서비스에 비즈니스 로직이 치중되있다고 생각이 들고 저 포문을 Symbols에다가 넣어야할까? 하고 고민이 되는데 Symbols의 용도는 그냥 기호들의 역할을 하는 클래스인데 넣어도되는걸까요?

Reply:
음 제 생각에 해당 서비스에 로직이 치중되어 있는 것 같진 않습니다. 다만, 각 일급 객체들에 대한 엔티티 그래프 탐색 깊이가 너무 깊은 것 같아요. 
```suggestion
for (int i = 0; i < symbols.size(); i++) {
            Arithmetic arithmetic = Arithmetic.fromSymbol(symbols.get(i));
            value = arithmetic.apply(value, numbers.get(i + 1));
        }
```

entity graph탐색에 필요한 비용은 각 일급 컬렉션에 맡겨도 될 것 같네요!